homeassistant/components/bloomsky/sensor.py

```diff
@@ -37,25 +35,28 @@
 
 # Sensor units - these do not currently align with the API documentation
 SENSOR_UNITS_IMPERIAL = {
-    "Temperature": TEMP_FAHRENHEIT,
+    "Temperature": UnitOfTemperature.FAHRENHEIT,
     "Humidity": PERCENTAGE,
-    "Pressure": PRESSURE_INHG,
+    "Pressure": UnitOfPressure.INHG,
     "Luminance": f"cd/{AREA_SQUARE_METERS}",
-    "Voltage": ELECTRIC_POTENTIAL_MILLIVOLT,
+    "Voltage": UnitOfElectricPotential.MILLIVOLT,
 }
 
 # Metric units
 SENSOR_UNITS_METRIC = {
-    "Temperature": TEMP_CELSIUS,
+    "Temperature": UnitOfTemperature.CELSIUS,
     "Humidity": PERCENTAGE,
-    "Pressure": PRESSURE_MBAR,
+    "Pressure": UnitOfPressure.MBAR,
     "Luminance": f"cd/{AREA_SQUARE_METERS}",
-    "Voltage": ELECTRIC_POTENTIAL_MILLIVOLT,
+    "Voltage": UnitOfElectricPotential.MILLIVOLT,
 }
 
 # Device class
 SENSOR_DEVICE_CLASS = {
     "Temperature": SensorDeviceClass.TEMPERATURE,
+    "Humidity": SensorDeviceClass.HUMIDITY,
+    "Pressure": SensorDeviceClass.PRESSURE,
+    "Voltage": SensorDeviceClass.VOLTAGE,
```

Comment:
Note: I didn't add the device class for `Luminance` because I am not sure if SensorDeviceClass.ILLUMINANCE is appropriate.